MAUI/Pull-to-Refresh/getting-started.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Getting Started with .NET MAUI PullToRefresh control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI PullToRefresh (SfPullToRefresh) control, its elements, and more.
+platform: MAUI
+control: SfPullToRefresh
+documentation: ug
+---
+
+# Getting Started with .NET MAUI PullToRefresh (SfPullToRefresh)
+
+This section provides a quick overview of how to get started with the .NET MAUI PullToRefresh (SfPullToRefresh) for MAUI. Walk through the entire process of creating the real-world SfPullToRefresh.
```

Comment:
and a walk-through to configure the .NET MAUI PullToRefresh control in a real-time scenario.

Reply:
Added review corrections